src/install.coffee

```diff
@@ -355,6 +355,10 @@ class Install extends Command
             @installNode (error) -> next(error, packagePath)
         commands.push (packagePath, next) =>
           @installModule(options, pack, packagePath, next)
+        if installGlobally and packageName isnt pack.name
+          commands.push (newPack, next) => # package was renamed; delete old package folder
+            fs.removeSync(path.join(@atomPackagesDirectory, packageName))
```

Comment:
I looked into it further and it looks to be having an issue with this line. More specifically the `fs.removeSync` although I can't seem to figure out why it is throwing an error.

Reply:
You can probably wrap this in a try/catch and log the error.

Reply:
Tried that. It doesn't display an error. It won't even let jasmine finish testing all the specs.

Reply:
Finally figured it out. The test-module2 fixture was giving the wrong name so it was failing on other tests.